src/Component/RuleTable/RuleTable.tsx

```diff
@@ -428,6 +445,11 @@ export class RuleTable extends React.Component<RuleTableProps, RuleTableState> {
     } = this.props;
 
     const columns: ColumnProps<RuleRecord>[] = [{
+      dataIndex: '',
```

Comment:
Just asking because it looks strange: As is this intended to be an empty string?

Reply:
Correct, `dataIndex` is optional so we can skip it completely. See https://github.com/ant-design/ant-design/blob/1bf0bab2a7bc0a774119f501806e3e0e3a6ba283/components/table/Column.tsx#L4